accounts/keystore/keystore.go

```diff
@@ -284,6 +284,23 @@ func (ks *KeyStore) SignTx(a accounts.Account, tx *types.Transaction, chainID *b
 	return nil, ErrChainIdNil
 }
 
+// SignTxAsFeePayer signs the given transaction as a fee payer with the requested account.
+func (ks *KeyStore) SignTxAsFeePayer(a accounts.Account, tx *types.Transaction, chainID *big.Int) (*types.Transaction, error) {
```

Comment:
```suggestion
func (ks *KeyStore) SignTxAsFeePayer(a accounts.Account, tx *types.Transaction, id *big.Int) (*types.Transaction, error) {
```

Reply:
I can't find where your naming rule is used. The variable name was copied from `SignTx`.
https://github.com/klaytn/klaytn/blob/dfb6591884fbdc2e86d56f9efc05cd39b004ad49/accounts/keystore/keystore.go#L271